back/engines/commercial/analysis/app/lib/analysis/nlp_cloud_helpers.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'nlpcloud'
+
+module Analysis
+  module NLPCloudHelpers
```

Comment:
Not sure whether there will be anyhthing else for Nlpcoud in our repo, so prefer to keep it relatively flat.

I was also thinking of making this a service instead of going with the mixin. It's easier to mock and more explicit to find back the method definitions from the caller code.

Reply:
> It's easier to mock and more explicit to find back the method definitions from the caller code.

It's about time you invest in a real editor 😜